### PR TITLE
Reviews: Style Transfer review, along with image assets.

### PR DESCRIPTION
**Pull request by cinjon**

### CODE REVIEW

**Comment by cinjon** [comment]
I signed it! (/ am a Googler)

**Comment by bryongloden** [comment]
@googlebot I am confused. I remember reading Googlers didn't need to sign the CLA

**Comment by douglaseck** [comment]
You are correct. Googlebot gets confused by personal + google.com email
addresses sometimes.. No harm in ACKing the agreement:-)

File: magenta/reviews/styletransfer.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+A Neural Algorithm of Artistic Style
+=====================================================
+
+   In late August 2015, Gatys et al. from The University of Tübingen published [A Neural Algorithm of Artistic Style](http://arxiv.org/pdf/1508.06576v2.pdf). It demonstrated a way to present one piece of artwork in the style of a separate piece and subsequently swept across Facebook walls around the world. In short, it captured the public’s attention and made them recognize that the tools we had been building for imaging applications could be used to create imaginative art.
+
+   So how did it work?
+
+<p align="center">
+  <img src="assets/tubingen-starry-night.jpg">
```
Comment (fredbertsch) [inline magenta/reviews/styletransfer.md]: Please add a note about copyright. (And for the other images as well.)

**Pull request merged**